### PR TITLE
Introduce UI translation change process checklist

### PR DESCRIPTION
**Pull request by fadani**

### CODE REVIEW

File: UI-translation-checklist.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## UI translation update
+
+- [ ] Devs add/modify the [UI spreadsheet](https://docs.google.com/spreadsheets/d/13q-eCtIJpnPSyAs9EjXo_fr3GxJt0sOEkSbcvwz6UmM/edit#gid=0) based on @thomasengebrand's mockups
+- [ ] Devs create a new [localization trello](https://trello.com/b/5g80uzTB/localization-content) card
```
Comment (anta-semenov) [inline UI-translation-checklist.md]: If Shannon need to check copy, will it make sense to add link to project doc (and name of a slack channel) to the trello card, so Shannon could easily find context of the strings?

Comment (fadani) [reply UI-translation-checklist.md]: Yes, at least mentioning the project is probably useful. I'm adding this

Comment (fadani) [inline UI-translation-checklist.md]:
```suggestion
- [ ] Devs create a new [localization trello](https://trello.com/b/5g80uzTB/localization-content) card with mentioning the project name/doc/channel
```

Comment (fadani) [reply UI-translation-checklist.md]: done

**Pull request merged**